demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -657,6 +662,16 @@ def run(
         # Detailed packages status
         pkgs_status: dict = {}
 
+        # Check docker image flags are in order
+        if (
+            docker_image_target
+            and docker_image_flag != DockerImageFlagOption.NATIVE_TARGET.value
+        ):
+            raise ValueError(
+                f"Recieved docker image target {docker_image_target} without docker "
+                f"image flag {DockerImageFlagOption.NATIVE_TARGET.value}. Aborting."
+            )
+
```

Comment:
What would be the implications of just assuming that `dit` implies `docker_image_flag`?

Reply:
not sure I understand, it would raise a value error like you can see from code section that is highlighted

Reply:
yeah, that's the existing behavior according to the code. We could decide that instead of raising an error, we'd just assume the other flag was used as well. What do you think about that?

Reply:
if one would specify --docker_image_flag "native:target" --dit "native:dev", no such image would be found in the registry

Reply:
in this case, we could raise an error, rather than block usage without the flag

Reply:
I see that it makes the first flag useless, but I don't think I want that, this can cause confusion between `--docker_image` and `--docker_image_target`
If I want to test a certain file (that is not native:dev supported) with a specified docker image, I could use --docker_image_target by mistake, causing it to run on an empty set of content items, and pass without me knowing this was never checked. 
if we use both, we guarantee the user knows a little bit more about what they are doing

that said, this is not such a strong argument & I can change this if you feel this is worth it